internal/operands/metrics/resources.go

```diff
@@ -45,7 +47,9 @@ func newPrometheusRule(namespace string) *promv1.PrometheusRule {
 								"runbook_url": "https://kubevirt.io/monitoring/runbooks/SSPOperatorDown",
 							},
 							Labels: map[string]string{
-								"severity": "critical",
+								"severity":                      "critical",
```

Comment:
Can you please extract severity as separate constant?

Reply:
Hi, @ksimon1, severity changes between alerts so we rather not using constant in the case.